tox.ini

```diff
@@ -92,3 +94,18 @@ allowlist_externals =
   rm
 commands =
   rm -rf {toxinidir}/docs/stubs/ {toxinidir}/docs/_build
+
+[testenv:tutorials]
+usedevelop = False
+basepython = python3
+setenv =
+  {[testenv]setenv}
+  RUST_DEBUG=1  # Faster to compile.
+  PYDEVD_DISABLE_FILE_VALIDATION=1
+deps =
+  {[testenv:docs]deps}
+  cvxpy
+  networkx>=2.3
+  tweedledum>=1.0.0,<2.0.0; python_version < '3.11' and platform_system != "Darwin"
```

Comment:
This works properly, even though GitHub renders the `;` as a comment

Reply:
Yeah, we can blame Microsoft for that one.  Or maybe blame Tox for using a file format with little-to-no actual specification...